conctr.agent.class.nut

```diff
@@ -114,41 +114,56 @@ class Conctr {
                 // Set the model
                 v._model <- _model;
 
-                // Set the time stamp if not set already
-                if (!("_ts" in v) || (v._ts == null)) {
+                local shortTime = false;
+                if (("_ts" in v) && (typeof v._ts == "number")) {
+                    // Invalid numerical timestamp? Replace it.
+                    if (v._ts < MIN_TIME) {
+                        shortTime = true;
+                    }
+                } else if (("_ts" in v) && (typeof v._ts == "string")) {
+                    // Invalid string timestamp? Replace it.
+                    if (v._ts.len() <= 10 && v._ts.tointeger() < MIN_TIME) {
```

Comment:
Won't this break if they send the ts as a string e.g. "21/10/2017" or in an iso8601 format?